config.md

```diff
@@ -38,7 +38,8 @@ For example, if a configuration is compliant with version 1.1 of this specificat
     On Linux, for example, with a bundle at `/to/bundle` and a root filesystem at `/to/bundle/rootfs`, the `path` value can be either `/to/bundle/rootfs` or `rootfs`.
```

Comment:
FWIW, I think these all should be sub-bullets.
  
o  On Windows, ...
o  On all other platforms, ...
o On Linux, ...
o If defined, ...